src/release.ts

```diff
@@ -51,3 +50,19 @@ function getAssetPrefix() {
   }
   return `ec-${platform}-${arch}`
 }
+
+// eslint-disable-next-line  @typescript-eslint/no-explicit-any
```

Comment:
I might have been a bit lazy here, feel free to correct me with a better idea :sweat_smile: